### ISSUE TITLE
Feature request: Keep Audacity minimized during import

### ISSUE THREAD

**Issue by goranbr** [audacity/audacity#4376]
When importing multiple files with audacity, it is really boring to sit and watch say 200 files get imported.

But if you try to minimize the application window while it imports, audacity refuses to stay minimized, and instead pops up the application windows full size after every single file is imported, like that game "whack-a-mole" at an amusement park. Again and again....  This can hardly be what most users want? 

Please, is it possible to change this behaviour to just let audacity stay minimized at least until **all** files selected for import is imported and work can begin? Or, even better, the application should just stay minimized until the user awakes it from the task bar to work with it again, like how all other applications behave.

I would really appreciate this change for future versions.

Version I use: Audacity 3.2.0
Platform: Kubuntu 22.10

**Comment by petersampsonaudacity** [comment]
Testing on W11 with 3.2.4 (and the beta test Ihave for 3.2.5) I find that I can have Audacity properly hidden behind other work that I am doing on my PC.  

In fact I have a full screen window of Firefox with me writing this note while Audacity is hidden and busty importing 65 3-minute or so song files.

**Comment by goranbr** [comment]
Yes, that's the behaviour most people would expect and prefer, I assume, including me.
So I guess this is specific behaviour for KDE, in this case, then?

I'm using Kubuntu 22.10, which uses KDE, and in this case with Wayland underneath.
I'll check if this is only for the Wayland graphics engine just to narrow down this quirky behaviour.

Somehow, in KDE  the application is sending a signal to the desktop environment after every file is imported like "Hey, I just finished importing one of the 100 files you asked for, ain't that great?" when it should probably hold off that until all files selected for import have been imported and say "Good news, I finished importing all the files you asked for." :-)

It doesn't even help to put the whole application in another virtual desktop. After every file it switches you back to the virtual desktop it uses, transitions from minimized state to full window and goes "Hey, dude, here's an update: I just imported one of the 100 files you needed! Great job huh? Please, stay tuned while I do this 99 more times".   :-) 

So, minimizing has lost it meaning...for now.

**Comment by goranbr** [comment]
**Update:**
The behaviour is not specific to KDE running on Wayland, but is seen also on standard Plasma running on Xorg:
That is: plasma-desktop 4:5.25.5-0ubuntu1

Thanks for updating the title to better reflect the issue on hand.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
